api/mysagw/identity/permissions.py

```diff
@@ -25,20 +33,16 @@ def has_object_permission(self, request, view, obj):
         return self.has_permission(request, view)
 
 
-class IsOrgAdmin(BasePermission):
-    def has_permission(self, request, view):
-        return True
-
+class IsAuthorized(BasePermission):
     def has_object_permission(self, request, view, obj):
-        return obj in request.user.identity.authorized_for
-
+        return obj in request.user.identity.authorized_for or (
+            getattr(obj, "identity", False)
```

Comment:
hm. `hasattr()`?

Reply:
Alternatively, seeing that we're going to use this check all over the place: It would probably be a good idea to abstract this away (getter property in `BaseUser` for example)

Reply:
Used `hasattr` now.

> Alternatively, seeing that we're going to use this check all over the place: It would probably be a good idea to abstract this away (getter property in BaseUser for example)

I think you missed, that in some places we check for the value of a property on the user (not available on `AnonymousUser`) and in other places we check for the existence of an `Identity` property on different objects.

Reply:
right!